app/src/main/java/net/skyscanner/backpack/demo/stories/ChipStory.kt

```diff
@@ -29,6 +30,7 @@ class ChipStory : Story() {
         forEachChip(child, cb)
       } else if (child is BpkChip) {
         cb(child)
+        child.chipBackgroundColor = ContextCompat.getColor(child.context, listOf(R.color.bpkBlue300, R.color.bpkBlue400, R.color.bpkBlue500, R.color.bpkBlue600, R.color.bpkBlue700, R.color.bpkBlue800, R.color.bpkBlue900).shuffled().first())
```

Comment:
Can we just set it from XML?

Reply:
Sorry. It was just for testing. Reverted